Login/sign up page UI

Youngphil5 opened the pull request:
Hey everyone, just finished the UI for the login and signup page. We should all try to have it in our codebase as soon as we can.

ae-ramirez reviewed (approved):
The design of the pages looks good.

Isabel-Kasim reviewed (approved):
Looks good to me, don't see any major issues yet. The design looks nice.

Youngphil5 commented:
Change list:

1. Deleted most of the default class:

-  first fragment class and its xml
- second fragment class and its xml
- the default main activities 

2. I added :

- a new mainActivity class and its xml
- a signup Activity class and its xml
- a login Activity class and its xml

3. I pulled in the commits from Isabel and Ramirez

4. I updated the Gradle dependencies for room because it gave me error on my end
